doc-General_Configuration/topics/Configuration.adoc

```diff
@@ -647,25 +647,27 @@ These roles are enabled by default on all servers.
-|SmartState Analysis|This role is enabled by default. The SmartState Analysis role controls which {product-title} servers can control SmartState Analyses and process the data from the analysis. You should have at least one of these in each zone.
-|User Interface|This role is enabled by default. Uncheck User Interface if you do not want users to be able to access this {product-title} server using the {product-title} console. For example, you may want to turn this off if the {product-title} server is strictly being used for capacity and utilization or reporting generation. More than one {product-title} server can have this role in a zone.
-|Web Services|This role is enabled by default. Uncheck Web Services to stop this {product-title} server from acting as a Web service provider. More than one {product-title} server can have this role in a zone.
+|Server Role|Description|Zone or Region Aware|Primary/Secondary or Active/Active
+|Automation Engine|Use this role if you want to use this {product-title} server to process automation tasks.|N/A|Active/Active
+|Capacity and Utilization Coordinator|The Capacity & Utilization Coordinator role checks to see if it is time to collect data, somewhat like a scheduler. If it is time, a job is queued for the Capacity and Utilization Data Collector. The coordinator role is required to complete Capacity and Utilization data collection. If more than one {product-title} server in a specific zone has this role, only one will be active at a time.|Zone|Primary/Secondary
+|Capacity & Utilization Data Collector|The Capacity & Utilization Data Collector performs the actual collection of capacity and utilization data. This role has a dedicated worker, and there can be more than one {product-title} server with this role in a zone.|Zone|Active/Active
+|Capacity & Utilization Data Processor|The Capacity & Utilization Data Processor processes all of the data collected, allowing {product-title} to create charts. This role has a dedicated worker, and there can be more than one {product-title} server with this role in a zone.|Zone|Active/Active
+|Database Operations|Use Database Operations to enable this {product-title} server to run database backups or garbage collection.|Zone|Active/Active
+|Event Monitor|This role is enabled by default and provides the information shown in timelines. Event Monitor is responsible for the work between the {product-title} server and your providers. It starts 2 workers for each provider. One worker, the monitor, is responsible for maintaining a connection to a provider, catching events, and putting them on the {product-title} message queue for processing. The second worker, the handler, is a message queue worker responsible for delivering only those messages for a provider. You should have at least one of these in each zone.|Zone|Primary/Secondary
```

Comment:
I think this would read better with a "The" in this sentence: "The Event Monitor is responsible for the work between the {product-title} server and your providers."